include/zephyr/bluetooth/gatt.h

```diff
@@ -736,6 +736,8 @@ struct bt_gatt_ccc_cfg {
 	uint8_t id;
 	/** Remote peer address. */
 	bt_addr_le_t peer;
+	/** Separate storage for encrypted and unencrypted context. */
+	bool link_encrypted;
```

Comment:
Does this indicate that the link was encrypted when the CCC was written, or does it indicate that the CCC requires encryption? 

IMO the documentation for this field is unclear

Reply:
It's the first option. I'll try to write a better documentation or maybe do it differently.

Reply:
Do it matter if the link was encrypted when the CCC was written?

Reply:
Hmm, but if a malicious device is spoofing the address, what is stopping the device from also encrypting the link?

Reply:
Nothing but if the good device already bonded, when the malicious device will try to restore the security, it will fail, the keys will not be correct.